x-pack/plugins/discover_log_explorer/common/constants.ts

```diff
@@ -0,0 +1,8 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+export const DISCOVER_LOG_EXPLORER_PROFILE_ID = 'discover-log-explorer';
```

Comment:
How about we change this to `log-explorer`? The `discover` part seems redundant as it will be part of the Discover URL or passed to the Discover locator anyway.